sdk/keyvault/keyvault-keys/test/public/keyClient.hsm.spec.ts

```diff
@@ -71,5 +72,84 @@ onVersions({ minVer: "7.2" }).describe(
         );
       });
     });
+
+    onVersions({ minVer: "7.3-preview" }).describe("releaseKey", () => {
+      const releasePolicy = {
+        anyOf: [
+          {
+            anyOf: [
+              {
+                claim: "sdk-test",
+                condition: "equals",
+                value: "true"
+              }
+            ],
+            // TODO: formalize this setup when deployment is automated.
```

Comment:
I need to explain this test setup - with the service team's help I created a mock service that pretends to be an attestation service. 

The managed HSM requires an attestation service that supports OIDC (OpenID Connect Discovery). The minimal requirements for us are:
- It should be reachable by the MHSM
- It should respond to `https://server.com/.well-known/openid-configuration` with the jwks_uri 
- the jwks_uri value should be reachable by the MHSM
- The attestation we provided must have the right information

The last bit is where it gets interesting - we need to use the same keys to sign the JWT as the ones returned by jwks_uri. 

So this should be something that is stood up by live test resources if we want to test this live. You can see what my mock service returns here:

https://malegeskr5.azurewebsites.net/generate-test-token -> used by us to get an attestation signed JWT
https://malegeskr5.azurewebsites.net/keys -> used by the MHSM to get the key used for signing the token
https://malegeskr5.azurewebsites.net/.well-known/openid-configuration -> tells the MHSM where to go to get key

Reply:
I'll investigate ways to make this part of our test resource deployment, but it's also possible to use this today to record tests for any language